pom.xml

```
@@ -179,6 +182,16 @@
                 <skip.unit.tests>${skipTests}</skip.unit.tests>
             </properties>
         </profile>
+        <!-- Till https://github.com/powermock/powermock/issues/1094 -->
+        <profile>
+            <id>openjdk17</id>
```

Suggest aligning the profile name with the version range.
```suggestion
            <id>java16+</id>
```